system/kube-monitoring/charts/prometheus-collector/aggregation.rules

```diff
@@ -1,6 +1,7 @@
 groups:
 - name: cpu
   rules:
+  # TODO: remove with K8S 1.10+ / cAdvisor 0.29.0 (https://github.com/google/cadvisor/blob/master/CHANGELOG.md#0290-2018-02-20)
```

Comment:
Note that kube-monitoring is also deployed to the scalout clusters running on k8s 1.10.
Shouldn't do any harm, right. Just no more per-cpu metrics in cAdvisor 0.29.0+

Reply:
sure, this is just a reminder to clean up once everything is on 1.10 or later